src/app/components/team/Newsletter/NewsletterRssFeed.js

```diff
@@ -87,12 +87,34 @@ const NewsletterRssFeed = ({
                     onBlur={(e) => {
                       if (e.target.value === '') {
                         onUpdateUrl(null);
+                      } else {
+                        handleLoad();
                       }
                     }}
                     value={localRssFeedUrl}
                     iconLeft={<LinkIcon />}
                     error={invalid}
-                    helpContent={helpContent || (invalid && <FormattedMessage id="newsletterRssFeed.error" defaultMessage="A valid RSS URL is required to load articles." description="Error message displayed under RSS feed URL field" />)}
+                    helpContent={
+                      helpContent ||
+                      (invalid &&
+                        <FormattedMessage
+                          id="newsletterRssFeed.error"
+                          defaultMessage="This URL is not a valid RSS URL. {helpLink}."
+                          description="Error message displayed under RSS feed URL field. The placeholder 'helpLink' is a link to a help article, and the text is something like 'Learn more'."
+                          values={{
```

Comment:
While this works, I'm not sure if there's any value in using a variable, this could be a couple of FormattedMessages separated by a &nbsp;

Reply:
Yeah this was one of the things I was not aware of... how would that work for Arabic and other languages for example?

Reply:
Hmmm. I'd expect it should just work :thinking:

Reply:
It's still text and first sentence should display before second sentence. FormattedMessage are rendered as span elements iinm, I expect it to just work...

Reply:
Because they're inline elements, right?

Reply:
Yeah I was not sure if for Arabic and other RTL languages the translation would be something like `{learnMore} text text text`.

Reply:
I think the translation is still `text text text {learnMore}` just rendered RTL. I.e. Learn more doesn't need to be typed first or re-located, it's still in the end of the string

Reply:
I pushed a change, please review.